exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/QaServiceImpl.java

```diff
@@ -89,8 +98,11 @@ protected Schema createDataSchema(View view) {
   @Override
   public Optional<String> initialize(Fork fork) {
     // Add a default counter to the blockchain.
-    String defaultCounterName = "default";
-    new CreateCounterTx(defaultCounterName)
+    new CreateCounterTx(DEFAULT_COUNTER_NAME)
```

Comment:
I propose to extract that in `createCounter(name: String)` method to hide irrelevant implementation details — the fact that we create transaction is a little misleading because it is not a regular transaction, we just want to reuse the code in its `#execute`.